de.ient.YUView.yaml

```diff
@@ -0,0 +1,68 @@
+app-id: de.ient.YUView
+runtime: org.kde.Platform
+runtime-version: 5.11
+sdk: org.kde.Sdk
+command: YUView
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=host
+  - --share=network
+  - --env=PATH=/usr/bin:/app/bin:/app/bin/x86_64-linux
+  - --env=QT_NO_FT_CACHE=1
+build-options:
+  env:
+    LD_LIBRARY_PATH: /app/lib:/app/lib64
```

Comment:
Why is this needed?

Reply:
```
- --env=QT_NO_FT_CACHE=1
```
is a workaround for a bug in QT which causes the application to crash sporadically. The rest can be removed, thanks.